base_konnector.js

```diff
@@ -62,7 +62,7 @@ module.exports = {
               resolve(account)
             })
             .catch(err => {
-              log('error', `error while getting the folder path from ID : "${folderId}"`)
```

Comment:
Actually, this detailled message was also interesting. Can you keep it as a debug message ?
```javascript
log('debug', `error while getting the folder path from ID : "${folderId}"`)
```